lib/codelog/clis/interactive.rb

```diff
@@ -0,0 +1,54 @@
+require 'thor'
+
+module Codelog
+  module CLIs
+    class Interactive < Thor
+      include Thor::Actions
+
+      no_commands do
+        def initialize
+          @sections = YAML.load_file('changelogs/template.yml').keys
+        end
+
+        def ask_for_changes
+          change_type = ask_for_type
+          say "\nType the entries for \e[33m#{change_type}\e[0m(ENTER to stop):"
+          result_hash = { change_type => build_array_response }
+          if yes? "\nWould you like to add a new log(Y|N)?"
+            result_hash.merge!(ask_for_changes) do |_, changes, changes_to_be_added|
+              changes | changes_to_be_added
```

Comment:
Maybe we should start using `+` instead of `|` for array merging. The pipe operator is associated to many different behaviors, I think it is kind of confusing to use it in this context

Reply:
For me, `+` is as bad as `|`. If you really want to change it, I suggest we use the [concat](https://apidock.com/ruby/Array/concat) method.

Reply:
I agree! The concat method is probably the best option! 👍